css/styles.css

```diff
@@ -329,6 +329,29 @@ tbody>td>.ordered-id {
     width: 225px;
 }
 
+#loadbar {
+    height: 6px;
```

Comment:
Nitpick, but could you put these style rules in alphabetical order? If you notice that it is incorrect in other places, feel free to change it too. 👍